pkg/reconciler/taskrun/taskrun.go

```diff
@@ -240,6 +248,16 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 		return err
 	}
 
+	if tr.IsSkipped() {
+		tr.Status.SetCondition(&apis.Condition{
+			Type:    apis.ConditionSucceeded,
+			Status:  corev1.ConditionFalse,
+			Reason:  "RunOnStateConflicted",
```

Comment:
Shouldn't it be `Reason:  resources.ReasonRunOnStateConflicted,` ?

Reply:
`ReasonRunOnStateConflicted` is declared in pipeline resources, should we introduce pipeline resources to taskrun resources? Not sure, might result in some conflicts down the road, may be, may not 🤔 OR declare it once again under taskrun resources and use it from there 😕